near-sdk-macros/src/lib.rs

```diff
@@ -27,7 +27,9 @@ pub fn near_bindgen(_attr: TokenStream, item: TokenStream) -> TokenStream {
             }
         };
         let generated_code = item_impl_info.wrapper_code();
+        let marshalled_code = item_impl_info.marshall_code();
```

Comment:
This may increase filesize unless it's used only for testing. Can we remove it from the wasm32 target?

Reply:
Yes it's only testing. It injects calls to ignore in wasm32 target. But it _should_ be here!  Thanks!